webservices/resources/dates.py

```diff
@@ -60,24 +58,25 @@ def get(self, **kwargs):
 @doc(description='FEC election dates since 1995.')
 class ElectionDatesView(DatesResource):
 
-    model = models.ElectionDates
+    model = models.ElectionDate
+
     @property
     def date_column(self):
         return self.model.election_date
 
-    filter_fields = {
-        'election_state',
-        'election_district',
-        'election_party',
-        'office_sought',
-        'election_date',
-        'trc_election_type_id',
-        'trc_election_status_id',
-        'update_date',
-        'create_date',
-        'election_year',
-        'pg_date',
-    }
+    filter_multi_fields = [
```

Comment:
Sounds good.

For reference, `pg_date` would be primary/general date. I am hoping that election type and status ids are going to lead to more data when we look at the new tables. They are probably not useful out of context. 

Thanks again!